site/_docs/reference.md

```diff
@@ -1824,33 +1824,34 @@ and `LISTAGG`).
 
```

Comment:
Can you revert the change in reference.md and then only add MODE documentation? If there is a need to re-order and fix existing function documentation, that should be done by another PR with its own JIRA. Let's keep this PR only for support MODE funciton.

Reply:
This change according to the JIRA code review:

- In reference.md, sort the table of aggregate functions into alphabetical order.

Reply:
Ah I see. Ok we can do it once then in the same PR.